egs/multi_zh_en/ASR/zipformer/multi_dataset.py

```diff
@@ -0,0 +1,242 @@
+        self.use_aishell2 = args.use_aishell2
+
+    def train_cuts(self) -> CutSet:
+        logging.info("About to get multidataset train cuts")
+
+        # AISHELL-2
+        if self.use_aishell2:
+            logging.info("Loading Aishell-2 in lazy mode")
+            aishell_2_cuts = load_manifest_lazy(
+                self.fbank_dir / "aishell2_cuts_train.jsonl.gz"
+            )
+
+        # TAL-CSASR
+        if self.use_tal_csasr:
+            logging.info("Loading TAL-CSASR in lazy mode")
+            tal_csasr_cuts = load_manifest_lazy(
+                self.fbank_dir / "tal_csasr_cuts_train_set.jsonl.gz"
+            )
+
+        # LibriSpeech
+        if self.use_librispeech:
+            logging.info("Loading LibriSpeech in lazy mode")
+            train_clean_100_cuts = self.train_clean_100_cuts()
+            train_clean_360_cuts = self.train_clean_360_cuts()
+            train_other_500_cuts = self.train_other_500_cuts()
+
+        if self.use_tal_csasr and self.use_librispeech and self.use_aishell2:
+            return CutSet.mux(
+                aishell_2_cuts,
+                train_clean_100_cuts,
+                train_clean_360_cuts,
+                train_other_500_cuts,
+                tal_csasr_cuts,
+                weights=[
+                    len(aishell_2_cuts),
+                    len(train_clean_100_cuts),
+                    len(train_clean_360_cuts),
+                    len(train_other_500_cuts),
+                    len(tal_csasr_cuts),
+                ],
+            )
+        elif not self.use_tal_csasr and self.use_librispeech and self.use_aishell2:
+            return CutSet.mux(
+                aishell_2_cuts,
+                train_clean_100_cuts,
+                train_clean_360_cuts,
+                train_other_500_cuts,
+                weights=[
+                    len(aishell_2_cuts),
+                    len(train_clean_100_cuts),
+                    len(train_clean_360_cuts),
+                    len(train_other_500_cuts),
+                ],
+            )
+        elif self.use_tal_csasr and not self.use_librispeech and self.use_aishell2:
+            return CutSet.mux(
+                aishell_2_cuts,
+                tal_csasr_cuts,
+                weights=[
+                    len(aishell_2_cuts),
+                    len(tal_csasr_cuts),
+                ],
+            )
+        elif self.use_tal_csasr and self.use_librispeech and not self.use_aishell2:
+            return CutSet.mux(
+                train_clean_100_cuts,
+                train_clean_360_cuts,
+                train_other_500_cuts,
+                tal_csasr_cuts,
+                weights=[
+                    len(train_clean_100_cuts),
+                    len(train_clean_360_cuts),
+                    len(train_other_500_cuts),
+                    len(tal_csasr_cuts),
+                ],
+            )
+        else:
+            raise NotImplementedError
```

Comment:
Please print the value of
- `self.use_tal_csasr`
- `self.use_librispeech`
- `self.use_aishell2`
on error.